public/app/plugins/datasource/elasticsearch/components/QueryEditor/BucketAggregationsEditor/SettingsEditor/DateHistogramSettingsEditor.test.tsx

```diff
@@ -1,42 +1,11 @@
-import { getDefaultTimeRange } from '@grafana/data';
```

Comment:
I'm wondering if it makes sense to have this test now. everything I was testing for here is now tested in `public/app/plugins/datasource/elasticsearch/components/hooks/useCreatableSelectPersistedBehaviour.test.tsx` and I feel this test is not adding any value, just artificially increasing the coverage

Reply:
I would be okay with removing it as well. As you've said, it is testing the same thing that is tested in `useCreatableSelectPersistedBehaviour`.